public/js2/kaiadmin.js

```
@@ -0,0 +1,406 @@
+	$(".form-group-default .form-control").focus(function(){
+		$(this).parent().addClass("active");
+	}).blur(function(){
+		$(this).parent().removeClass("active");
+	})
+
+});
+
+// Input File Image
+
+function readURL(input) {
+	if (input.files && input.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)